src/config/scrivitoSites.ts

```diff
@@ -20,19 +26,27 @@ export function baseUrlForSite(siteId: string): string | undefined {
 export function siteForUrl(
   url: string,
 ): { baseUrl: string; siteId: string } | undefined {
+  const neoletterMailingsBaseUrl = baseUrlForSite(NEOLETTER_MAILINGS_SITE_ID)
+  if (neoletterMailingsBaseUrl && url.startsWith(neoletterMailingsBaseUrl)) {
```

Comment:
Yes, the way we currently type `baseUrlForSite` makes TS believe it can be undefined for NEOLETTER_MAILINGS_SITE_ID. Alternatives:
* Force bang
* Force bang and comment
* Invest in typing baseUrlForSite more thoroughly

Reply:
Ok. Then let's leave it like it is.